test/controllers/name_controller_test.rb

```diff
@@ -3445,6 +3507,14 @@ def test_email_tracking_disable
   end
 
   # ----------------------------
+  #  Mushroom App
+  # ----------------------------
+
+  def test_names_for_mushroom_app
```

Comment:
Perhaps it's something Ray was talking about, but put on hold because MO now works so well responsively.  I'll ask him.  It shouldn't make any difference for this PR, but perhaps we can strip out some code at a later date.